Components/Resamplers/MyStandardResampler/elxMyStandardResampler.h

```diff
@@ -73,7 +73,8 @@ class MyStandardResampler :
   typedef typename Superclass1::InterpolatorPointerType InterpolatePointerType;
   typedef typename Superclass1::SizeType                SizeType;
   typedef typename Superclass1::IndexType               IndexType;
-  typedef typename Superclass1::PointType               PointType;
+  typedef typename Superclass1::InputPointType          InputPointType;
+  typedef typename Superclass1::OutputPointType         OutputPointType;
```

Comment:
Thank you Andras. I encountered the same problem, when trying to compile elastix with the latest revision from the master branch of ITK:
> error C2039: 'PointType': is not a member of 'itk::ResampleImageFilter'

Personally I would prefer to simply remove the `MyStandardResampler::PointType` typedef, because that could allow supporting _both_ ITK 5.0.1 and 5.1. (At least for now.) Do you think it's really necessary to add those `InputPointType` and `OutputPointType` typedefs here?

Reply:
I don't like having tons of typedefs but this is the ITK style. You can make it work with various ITK versions by using ifdefs.

Reply:
@lassoan I'm glad you don't like having tons of typedefs either 😃 Some typedefs could be useful, of course, so my main question would be: could anything be broken by removing those specific typedefs from `elastix::MyStandardResampler`. Especially because I feel that supporting multiple ITK versions by means of ifdefs is also a pain.

Now it looks like even without explicitly specifying those two typedef's with this PR, it's still possible to use `elastix::MyStandardResampler<T>::InputPointType` and `elastix::MyStandardResampler<T>::OutputPointType`, when using ITK 5.1rc03!!! Because such typedefs are publicly made accessibly already by its first base class, `itk::ResampleImageFilter`! So I don't think removing those typedefs from the PR does any harm.